lbry/extras/daemon/json_response_encoder.py

```diff
@@ -126,6 +127,8 @@ def default(self, obj):  # pylint: disable=method-hidden,arguments-differ,too-ma
             return self.encode_wallet(obj)
         if isinstance(obj, ManagedStream):
             return self.encode_file(obj)
+        if isinstance(obj, TorrentSource):
+            return self.encode_file(obj)
```

Comment:
`isinstance()` takes a tuple as second argument, so it could just be: `isinstance(obj, (ManagedStream, TorrentSource))`